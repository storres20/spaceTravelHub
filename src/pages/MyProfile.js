import React from 'react';
import { useSelector, useDispatch } from 'react-redux';

const MyProfile=()=> {

  const { rockets } = useSelector((state) => state.rockets);


  return (
    <div>MyProfile</div>
  );
}

export default MyProfile;
