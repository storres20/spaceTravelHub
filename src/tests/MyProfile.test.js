import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
// Redux
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
// ****
import MyProfile from '../pages/MyProfile';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <BrowserRouter>
          <MyProfile />
        </BrowserRouter>
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
