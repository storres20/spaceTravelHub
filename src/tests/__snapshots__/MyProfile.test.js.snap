// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container-fluid"
>
  <div
    className="justify-content-around flex-nowrap row"
  >
    <div
      className="col-md-6"
      style={
        Object {
          "width": "40%",
        }
      }
    >
      <div
        className="card border-light"
      >
        <div
          className="card-body"
        >
          <div
            className="card-title h5"
          >
            My Missions
          </div>
          <p
            className="card-text"
          >
            <ul
              className="list-group"
            />
          </p>
        </div>
      </div>
    </div>
    <div
      className="col-md-6"
      style={
        Object {
          "width": "40%",
        }
      }
    >
      <div
        className="card border-light"
      >
        <div
          className="card-body"
        >
          <div
            className="card-title h5"
          >
            My Rockets
          </div>
          <p
            className="card-text"
          >
            <ul
              className="list-group"
            />
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
